apps/envited-x-data-space/modules/DashboardNavigation/DashboardNavigation.tsx

```diff
@@ -15,21 +23,40 @@ import { ColorScheme } from '../../common/types'
 interface NavItemProps {
   href: string
   name: string
-  icon?: JSX.Element | string
+  icon?: boolean | JSX.Element
```

Comment:
Either a boolean or a react element? That seems like an anti-pattern? Why cant it always be either an element or the name of an element?